src/Controls/tests/Xaml.UnitTests/MSBuild/_Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <MauiRootDirectory>$(MSBuildThisFileDirectory)..\..\..\..\..\..\..\..\..\</MauiRootDirectory>
+    <MauiRootDirectory>$(MSBuildThisFileDirectory)..\..\..\..\..\..\..\</MauiRootDirectory>
```

Comment:
Is this changed because the output is no longer the bin in the folder but at the root?

Reply:
yeah the bin directory is now somthing like artifacts/bin/Project/Debug for example , i think we need to create a new task to redo these tests that use like thes strong typed paths?